src/modules/bootloader/main.py

```diff
@@ -403,6 +403,22 @@ def install_secureboot(efi_directory):
         "-p", efi_partition_number,
         "-l", install_efi_directory + "/" + install_efi_bin])
 
+    boot_entry = None
+    ret = subprocess.run([libcalamares.job.configuration["efiBootMgr"]],
```

Comment:
subprocess.run() is Python >=3.5, we want to stay compatible with Python 3.3, so you cannot use it.

Reply:
So I guess, the code should be:

```
    boot_entry = None
    efi_entries = subprocess.check_output([libcalamares.job.configuration["efiBootMgr"]], text=True)
    for line in efi_entries.split('\n'):
        if not line:
            continue
        words = line.split()
        if len(words) >= 2 and words[0] == "BootOrder:":
            boot_entry = words[1].split(',')[0] 
            break
    if boot_entry:
        subprocess.call([
            libcalamares.job.configuration["efiBootMgr"],
            "-n", boot_entry])
```